opennms-model/src/main/java/org/opennms/netmgt/model/OnmsAcknowledgment.java

```diff
@@ -121,25 +121,36 @@ public OnmsAcknowledgment(final Event e) throws ParseException {
         
         for (Parm parm : parms) {
             final String parmValue = parm.getValue().getContent();
-            
-            if (!"ackType".equals(parm.getParmName()) && !"refId".equals(parm.getParmName()) && !"user".equals(parm.getParmName()) ) {
+            if (!"ackAction".equals(parm.getParmName()) && 
+                !"ackType".equals(parm.getParmName()) && !"refId".equals(parm.getParmName()) && !"ackUser".equals(parm.getParmName()) ) {
```

Comment:
Should we also continue to support the `user` parameter as well?

Reply:
Sure!

Reply:
Good Point